.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: 'v3.4.0'
+    hooks:
+      - id: trailing-whitespace
+
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: 'v7.21.0'
+    hooks:
+      - id: eslint
+
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: [--check]
```

Comment:
I would remove this and opt for a more heavy handed approach. If users don't want it to automatically change things, they can either remove this line locally or just run it manually.

Reply:
Removed